IPython/core/excolors.py

```diff
@@ -155,6 +156,12 @@ def exception_colors():
         Normal = C.Normal,
         ))
 
+    # Hack: the 'neutral' colours are not very visible on a dark background on
+    # Windows. Since Windows command prompts have a dark background by default, and
+    # relatively few users are likely to alter that, we will use the 'Linux' colours,
+    # designed for a dark background, as the default on Windows.
+    if os.name == "nt":
+        ex_colors.add_scheme(ex_colors['Linux'].copy('Neutral'))
```

Comment:
I haven't use Windows in a while. I know there've been changes in the console, which is now white background (right) ?. This is IIRC one of the reason of "Neutral" to work both on Light and Dark Background.

But I'll trust a fellow windows user in the choice of colors. I've also tested locally and while it's not the coloring scheme I'm used too (put I'm picky and can have my own). I'm not opposed to making it the default even on non-'nt'